docs_app/content/deprecations/multicasting.md

```diff
@@ -34,7 +34,8 @@ Instead of creating a [ConnectableObservable](/api/index/class/ConnectableObserv
 
 <!-- prettier-ignore -->
 ```ts
-import { ConnectableObservable, Subject, timer } from 'rxjs';
```

Comment:
I'm not sure why this was changed? Was there some sort of automated fix?

Reply:
I don't use any tool, all I use is WebStorm and [this handful shortcut](https://www.jetbrains.com/webstorm/guide/tips/move-elements-left-right/) which works in this line as well, which is amazing, since this is not actual code, but a comment.

But, why I changed this line? Because I did the same set of changes in examples below this one - as I said in #6678, it was much easier for me to keep track of all objects I need. What I did with every changed example is to run the docs script, wait to re-render the example in browser and then I opened StackBlitz to verify that everything is working just fine. In the end, when SB opened and was ready, I checked every imported object to see if it is used - those that were not used were never highlighted in the code below - this is how ordering objects by the time they were used helped me.

In this particular example, it was just my OCD to make things consistent through entire file, nothing else. I don't ever expect anyone else to keep doing the same consistency changes as I did, sorry about that.